doc/sphinxman/source/scf.rst

```diff
@@ -925,15 +925,17 @@ Effective core potentials (ECPs)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 |PSIfour| supports the use of effective core potentials to describe the
-innermost electrons in heavy elements.  If a basis set is designed to use an
+innermost electrons in heavy elements.
+ECPs are only available if |PSIfour| is compiled with the :ref:`LibECPInt <cmake:ecpint>` library.
+If a basis set is designed to use an
 effective core potential, the ECP definition should be simply placed alongside
 the orbital basis set definition, *c.f.* :ref:`sec:basissets-ecps`.  All
 information related to the definition and number of core electrons will
 automatically be detected and no further input is required to use the
 ECP-containing basis set.  See :srcsample:`scf-ecp` and :srcsample:`dfmp2-ecp`
 for examples of computations with ECP-containing basis sets.
 
-.. warning:: Analytic derivatives of ECPs are not yet available.  The HF and DFT derivatives are implemented in a semi-numerical scheme, where numerical ECP gradients are added to analytic SCF gradients.  Analytic gradients for (DF)MP2 are not yet available, but the standard numerical gradients will work correctly.  Fully analytic gradients will be implemented soon.
+.. warning:: As of May 2022, v1.6, the LibECPInt library is used, and analytic derivatives and Hessians of ECPs are available. Previously, built-in code was used for ECPs, and analytic derivatives were not available. Then, the HF and DFT derivatives were implemented in a semi-numerical scheme, where numerical ECP gradients are added to analytic SCF gradients.  Analytic gradients for (DF)MP2 were not yet available, but the standard numerical gradients will work correctly.
```

Comment:
I don't quite understand the mixed tense (were / will) in the last sentence. I think this might read better if split in two:

.. warning:: Prior to May 2022, v1.6, Psi4 used a built-in ECP code. Analytic derivatives of ECPs were not  available. Numerical ECP gradients were added to analytic SCF gradients.

As of May 2022, v1.6, Psi4 uses the LibECPInt library, and analytic derivatives and Hessians of ECPs are available. [statement about mp2]

Reply:
tense is vague b/c I wasn't sure what the objective was (mp2 only, all post-scf, something else). right now, mp2 grad with ECP will fail b/c there's a Python catch. if that's removed, a quick mp2 and ccsd grad shows ECP gradients working in general. confirmed with Andy that this is expected. But this should go on a follow-up issue for testing.

Reply:
see how you like this wording.

```
.. warning:: Prior to May 2022, v1.6, Psi4 used a built-in ECP code. Analytic derivatives of ECPs were not available. The HF and DFT derivatives were implemented in a semi-numerical scheme, where numerical ECP gradients were added to analytic SCF gradients. For post-SCF methods, the entire gradient computation needed to be run as finite difference of energies.

.. warning:: As of May 2022, v1.6, Psi4 uses the LibECPInt library, and analytic derivatives and Hessians of ECPs are available. Analytic derivatives of molecular systems including ECPs should be available whenever the method has analytic derivatives, but these have so far only been verified for HF and DFT.
```

Reply:
Very clear, thank you!